gatsby/content/blog/2022/01/2022-01-25-synapse-1.51-released.mdx

```diff
@@ -0,0 +1,45 @@
+---
+date: '2022-01-25'
+title: Synapse 1.51 released
+categories:
+- Releases
+author: Brendan Abolivier
+---
+
+[Synapse 1.51](https://github.com/matrix-org/synapse/releases/tag/v1.51.0) is out! Here's what's new with this week's release.
+
+
+## Deprecation of the `webclient` listener
+
+A long time ago, Synapse used to serve a very basic web Matrix client (named "console") that could be used to connect to the homeserver. Server administrators could chose to make it available to their users by configuring a `webclient` listener.
+
+This web client was removed in [Synapse 0.34](https://github.com/matrix-org/synapse/releases/tag/v0.34.0) back in 2018, but the `webclient` listener stayed, instead allowing server admins to serve the web client of their choice (or redirect to it) through the `web_client_location` configuration file.
+
+This version of Synapse deprecates the `webclient` listener, as well as the ability to set `web_client_location` to a static directory (instead of a HTTP(S) URL). These will be removed in Synapse 1.53. See the [upgrade notes](https://matrix-org.github.io/synapse/develop/upgrade#upgrading-to-v1510) for more information.
+
+
+## Aggregations and sync
+
+The concept of a server-side aggregation in Matrix is defined in [MSC2675](https://github.com/matrix-org/matrix-doc/pull/2675) and is the ability for homeservers to extend the information included in an event using other events that relate to it. This allows, for example, clients to quickly retrieve the reactions associated with a given message, or its latest edit.
```

Comment:
```suggestion
The concept of server-side aggregation in Matrix is defined in [MSC2675](https://github.com/matrix-org/matrix-doc/pull/2675) and is the ability for homeservers to extend the information included in an event using other events that relate to it. This allows, for example, clients to quickly retrieve the reactions associated with a given message, or its latest edit.
```

nitpick: aggregation is both a verb and a noun, but colloquially I believe that it's better known as a verb (and the rest of the sentence refers to it as such).